.github/CONTRIBUTING.md

```diff
@@ -70,50 +70,121 @@ and use American spellings to denote classes, methods, fields, etc.
   public Color fontColor;
   ```
 
+All core classes should be within the relevant `VRTK` namespace. Any
+required `using` lines should be within the namespace block.
+
+  > **Incorrect:**
+  ```
+  using System;
+  namespace X.Y.Z
+  {
+    public class MyClass
+    {
+    }
+  }
+  ```
+
+  > **Correct:**
+  ```
+  namespace X.Y.Z
+  {
+    using System;

```

Comment:
Add a newline after this to ensure folks understand that part of our conventions?